perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -7476,6 +7476,21 @@ protected void initialize() {
 		rights.add(new AttributeRights(-1, Role.GROUPADMIN, Collections.singletonList(ActionType.READ)));
 		attributes.put(attr, rights);
 
+		//urn:perun:group:attribute-def:def:groupStructureResources
+		attr = new AttributeDefinition();
```

Comment:
Maybe we should start to call this part of the code from the module (if exists) to prevent duplicities.

Reply:
Good idea, I have thought of that as well.